src/microbe_stage/Microbe.cs

```diff
@@ -2582,6 +2589,11 @@ private void BeginBind()
             return;
         }
 
+        // A frame has passed since last check, so we get sure nothing got changed in between.
+        // We don't pass other to the *invoked* method for computational purposes.
+        if (!CanBindToMicrobe(other))
```

Comment:
Merge this with `var other` on line 2578 too.

Reply:
The thing is that we have error printing above, so we may want to keep them for defensiveness. However, it may in fact come in handy to avoid wasting a binding frame... 

I suppose I'll do the change because the latter seems outweight the former in terms of benefices.